frontend/src/components/course/components/choose-mentor-button/choose-mentor-button.tsx

```diff
@@ -0,0 +1,30 @@
+import chooseMentorImage from 'assets/img/choose-mentor-image.png';
+import { FC } from 'common/types/types';
+import { Button, Image } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  onClick: (evt: React.MouseEvent) => void;
+};
+
+const ChooseMentorButton: FC<Props> = ({ onClick }) => {
+  return (
+    <div className={styles.chooseMentorContainer}>
+      <div className={styles.image}>
+        <Image
+          alt="course image"
+          src={chooseMentorImage}
+          width="80%"
+          height="80%"
```

Comment:
```suggestion
          width="80"
          height="80"
```
do it with styles. on the html tags only this utis work